svg/elements/textpath.json

```diff
@@ -302,10 +302,10 @@
                 "version_added": true
               },
               "edge": {
-                "version_added": null
+                "version_added": true
```

Comment:
This should actually be `12`, as well as everywhere else where it currently says `true` for Edge (and probably Edge mobile, too).